pkg/client/errcat/errors.go

```diff
@@ -0,0 +1,68 @@
+package errcat
+
+import (
+	"errors"
+	"fmt"
+)
+
+// The Category is used for categorizing errors so that we can know when
+// to point the user to the logs or not.
+type Category int
+
+type categorized struct {
+	error
+	category Category
+}
+
+const (
+	OK       = Category(iota)
+	User     // User made an error
+	Config   // Errors in config.yml, extensions, or kubeconfig
+	OtherCLI // Error generated in the CLI process, so no use pointing logs
```

Comment:
Again this is kind of a nit, but I think perhaps a better name for this would be `NoLogs`, it took me a bit to grok what `OtherCLI` meant, and I think it'd be even harder without the context of this PR.